### PR TITLE
Landing product cat product list

### PR DESCRIPTION
**Pull request by j24m**

### CODE REVIEW

File: src/components/productCard/Product_card.css [hunk 1]
```diff
@@ -2,13 +2,49 @@
 
 .product-card-component .e-commerce-card {
   color: var(--secondary-color);
-  height: 45rem;
-  width: 32rem;
-  padding: 2rem;
+  color: var(--text-gray-color);
+  position: relative;
+  box-shadow: 0px 2px 4px var(--text-gray-color),
```
Comment (srishtigautam21) [inline src/components/productCard/Product_card.css:7]: no need to give unit px if value is zero

File: src/backend/db/products.js [hunk 2]
```diff
@@ -7,28 +7,201 @@ import { v4 as uuid } from "uuid";
...
+    title: "Glorious White Aura Keycaps",
+    brand: "Glorious Model",
+    rating: 3.2,
+    discount: 20,
+    price: 2599.0,
+    category: "Key Caps",
+  },
+  {
+    _id: 17,
+    image_url:
```
Comment (srishtigautam21) [inline src/backend/db/products.js:186]: use uuid for id

File: src/components/productCard/Product_card.jsx [hunk 3]
```diff
@@ -2,51 +2,62 @@ import "./Product_card.css";
...
+            {outOfStock ? (
+              <div className="disable-card">
+                <h3 className="outofstock-badge">Out Of Stock</h3>
               </div>
+            ) : (
+              <div className="badge-on-card">{discount}% Off</div>
+            )}
+            {/* <div className="btn-dismiss">
+              <MakiCross11></MakiCross11>
+            </div> */}
```
Comment (srishtigautam21) [inline src/components/productCard/Product_card.jsx:23]: Remove commented code

**Review by srishtigautam21 — APPROVED** [review]
The code looks clean
Suggestion:
Give some description to your PR like what feat you have added in this PR and the files to check so that it becomes easier for a reviewer to go through your code

File: src/pages/product_listing_page/Product_listing_page.jsx [hunk 4]
```diff
@@ -4,16 +4,27 @@ import ProductCard from "../../components/productCard/Product_card.jsx";
...
 
 const Products = (props) => {
+  const [products, setProducts] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const response = await fetch("/api/products");
+      const productData = await response.json();
+      setProducts(productData.products);
+    })();
+  }, []);
```
Comment (samyakshah3008) [inline src/pages/product_listing_page/Product_listing_page.jsx:18]: You can call APIs in separate file and import it here for cleaner code

**Review by samyakshah3008 — APPROVED** [review]
Added some suggestions in single comment, also found that you have not used css variables at few places, so do refactor it later on 
Rest is all good for approval 
Superb work!

**Pull request merged**